### PR TITLE
Fix deploy scripts

### PR DESCRIPTION
**Pull request by gdbelvin**
Depends on #1330

### CODE REVIEW

File: scripts/deploy.sh [hunk 1]
```diff
@@ -35,6 +35,7 @@ gcloud --quiet auth configure-docker
 
 
 echo "Generating keys..."
+rm -f ./genfiles/*
```
Comment (mhutchinson) [inline scripts/deploy.sh]: Before running a command like this, it seems prudent to check/enforce the current directory is what you expect it to be.

**Comment by gdbelvin** [comment]
Added some checks -- not sure if they are rigorous enough though. 
Key Transparency can be checked out in any directory now that Go modules are a thing. This makes creating strong checks around what directory the script expects to be run in more difficult.

File: scripts/deploy.sh [hunk 2]
```diff
@@ -33,8 +33,11 @@ gcloud --quiet config set container/cluster ${CLUSTER_NAME_CI}
 gcloud --quiet container clusters get-credentials ${CLUSTER_NAME_CI}
 gcloud --quiet auth configure-docker
 
+# Test current directory before deleting anything
+test $(basename $(pwd)) == "keytransparency" || exit
```
Comment (pavelkalinnikov) [inline scripts/deploy.sh]: Should this exit with a non-zero code?

Comment (gdbelvin) [reply scripts/deploy.sh]: Done

**Pull request merged**